src/components/Title.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Title = ({ text, tag: CustomTag }) => (
+        <div className="app__title">
```

Comment:
we need to think, and make a trial for removing this div, and style directly the title ;)